src/CommunityToolkit.Maui.Core/Views/DrawingView/PlatformView/MauiDrawingViewExtensions.shared.cs

```diff
@@ -0,0 +1,77 @@
+using System.Collections.ObjectModel;
+using CommunityToolkit.Maui.Core.Views;
+
+namespace CommunityToolkit.Maui.Core.Extensions;
+
+/// <summary>
+/// Extension methods to support <see cref="IDrawingView"/>
+/// </summary>
+static class MauiDrawingViewExtensions
```

Comment:
This should be public... We don't hold secrets on `Core`
```suggestion
public static class MauiDrawingViewExtensions
```